src/bundler.ts

```diff
@@ -46,6 +47,7 @@ export interface Options {
   // or other hint to fix the top-level folder.
   basePath?: string;
 
+  // TODO(usergenic): Added Imports is not yet supported.
```

Comment:
And upon further further review it is clear that as defined, the `--add-import` option has no means to correctly relate to which bundle the added import would even be attached.  I've removed the option for now because:

 1. There's no clear way to actually implement it as expressed with current option structure.
 2. The result of adding import is surprising and contrasts with intended behavior of bundling.